docs/src/main/tut/faq.md

```diff
@@ -16,7 +16,7 @@ section: "faq"
  * [What does `macro Ops` do? What is `cats.macros.Ops`?](#machinist)
  * [How can I help?](#contributing)
 
-## What imports do I need?<a id="what-imports" href="#what-imports"></a>
+## <a id="what-imports" href="#what-imports"></a>What imports do I need?
```

Comment:
Hey @ceedubs - so the reason I switched the order, was that when I tested locally, when the anchor was defined after the text, I built locally and tested the page, and the problem was that clicking the link brought me to just after the heading. So, as the page is a bit larger now, that actually meant that the heading was not visible, and I had to scroll in reverse to see the heading.  I thought this would be confusing for anyone who clicked a link not to be able to see the heading of the link they clicked on so I switched the anchor to earlier. I agree about the anchor, but I think it might actually be worse not to be able to see the heading.